### ISSUE TITLE
Import YouTube playlist

### ISSUE THREAD

**Issue by dvirtz** [RustyNova016/MusicBrainz-UserScripts#9]
Sometimes albums are released as a YouTube playlist.
Can you support importing the playlist as a release?

**Comment by jdaviescoates** [comment]
Yeah, I'd really like this feature too! I was hoping that is what the script would do it seems not.
